### PR TITLE
Opera egis updates

### PR DESCRIPTION
**Pull request by jacquelynsmale**

### CODE REVIEW

File: image_services/opera/config/rtc_vv.json [hunk 1]
```diff
@@ -11,12 +13,12 @@
...
-    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='copyright']/Value": "OPERA Level-2 Radiometric Terrain Corrected Static Layers from Sentinel-1. Contains modified Copernicus Sentinel data, processed by ESA.",
+    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='copyright']/Value": "OPERA Level-2 Radiometric Terrain Corrected from Sentinel-1. NASA Alaska Satellite Facility Distributed Active Archive Center. doi: https://doi.org/10.5067/SNWG/OPERA_L2_RTC-S1_V1. Contains modified Copernicus Sentinel data, processed by ESA.",
```
Comment (AndrewPlayer3) [inline image_services/opera/config/rtc_vv.json:21]:
```suggestion
    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='copyright']/Value": "OPERA Level-2 Radiometric Terrain Corrected products from Sentinel-1. NASA Alaska Satellite Facility Distributed Active Archive Center. doi: https://doi.org/10.5067/SNWG/OPERA_L2_RTC-S1_V1. Contains modified Copernicus Sentinel data, processed by ESA.",
```

File: image_services/opera/config/rtc_vv_urls.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "Configurations/SVCConfiguration/Definition/Description": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL).",
+    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='description']/Value": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL). Values for the VV polarization are commonly driven by surface roughness and/or soil moisture, with rougher surfaces and higher soil moisture returning higher backscatter values. Surface water appears very dark under calm conditions, as the signal bounces off the surface away from the sensor.",
+    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='copyright']/Value": "OPERA Level-2 Radiometric Terrain Corrected from Sentinel-1. NASA Alaska Satellite Facility Distributed Active Archive Center. doi: https://doi.org/10.5067/SNWG/OPERA_L2_RTC-S1_V1. Contains modified Copernicus Sentinel data, processed by ESA.",
```
Comment (AndrewPlayer3) [inline image_services/opera/config/rtc_vv_urls.json:21]:
```suggestion
    "Configurations/SVCConfiguration/Definition/ConfigurationProperties/PropertyArray/PropertySetProperty[Key='copyright']/Value": "OPERA Level-2 Radiometric Terrain Corrected products from Sentinel-1. NASA Alaska Satellite Facility Distributed Active Archive Center. doi: https://doi.org/10.5067/SNWG/OPERA_L2_RTC-S1_V1. Contains modified Copernicus Sentinel data, processed by ESA.",
```

File: image_services/opera/make_opera_services.py [hunk 3]
```diff
@@ -10,22 +10,20 @@
...
-    rasters = []
-    s3 = boto3.client('s3')
-    paginator = s3.get_paginator('list_objects_v2')
-    for page in paginator.paginate(Bucket=bucket, Prefix=prefix):
-        for obj in page['Contents']:
-            if obj['Key'].endswith(suffix):
-                rasters.append(f'/vsis3/{bucket}/{obj["Key"]}')
-    return rasters
+
+def get_rasters(dataset_name):
```
Comment (AndrewPlayer3) [inline image_services/opera/make_opera_services.py:22]:
```suggestion
def get_rasters(dataset_name: str) -> List[str]:
```

File: image_services/opera/config/rtc_vv.json [hunk 4]
```diff
@@ -29,11 +31,10 @@
...
     "ItemInfo/Description": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL). Values for the VV polarization are commonly driven by surface roughness and/or soil moisture, with rougher surfaces and higher soil moisture returning higher backscatter values. Surface water appears very dark under calm conditions, as the signal bounces off the surface away from the sensor.",
     "ItemInfo/Snippet": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL).",
     "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension[TypeName='WMSServer']/Props/PropertyArray/PropertySetProperty[Key='title']/Value": "OPERA_RTC_VV_WMS",
     "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension[TypeName='WMSServer']/Props/PropertyArray/PropertySetProperty[Key='abstract']/Value": "WMS service for Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL)."
   }
-}
-
+}
```
Comment (jhkennedy) [inline image_services/opera/config/rtc_vv.json:40]:
```suggestion
}

```

File: image_services/opera/config/rtc_vv_urls.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    "ItemInfo/Description": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL). Values for the VV polarization are commonly driven by surface roughness and/or soil moisture, with rougher surfaces and higher soil moisture returning higher backscatter values. Surface water appears very dark under calm conditions, as the signal bounces off the surface away from the sensor.",
+    "ItemInfo/Snippet": "OPERA Level-2 Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL).",
+    "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension[TypeName='WMSServer']/Props/PropertyArray/PropertySetProperty[Key='title']/Value": "OPERA_RTC_VV_WMS",
+    "Configurations/SVCConfiguration/Definition/Extensions/SVCExtension[TypeName='WMSServer']/Props/PropertyArray/PropertySetProperty[Key='abstract']/Value": "WMS service for Radiometric Terrain Corrected (RTC) products in VV polarization generated from Sentinel-1 SAR imagery, processed by OPERA (JPL)."
+  }
+}
```
Comment (jhkennedy) [inline image_services/opera/config/rtc_vv_urls.json:40]:
```suggestion
}

```

File: image_services/opera/metadata/metadata_text/OPERA_RTC_CA_VH_metadata.txt [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+OPERA RTC-S1 data is available over all landmasses (excluding Antarctica) for Sentinel-1 acquisitions starting October 4, 2023. This is a test service that only includes the data available for a region in Northern California, acquired between October 7 and December 3, 2023. We are working towards providing services that include the full global extent of the dataset, but in incremental developments.
+
+The source rasters referenced in this service are hosted by the Alaska Satellite Facility in NASA's Earthdata Cloud. More information about the dataset is available here: [https://asf.alaska.edu/datasets/daac/opera/](https://asf.alaska.edu/datasets/daac/opera/)
+
+OPERA includes partners from NASA’s Goddard Space Flight Center, the U.S. Geological Survey (USGS), the University of Maryland College Park, the University of Alaska Fairbanks, Southern Methodist University, and EarthScope. Refer to [OPERA’s website](https://www.jpl.nasa.gov/go/opera) for more details.
```
Comment (jhkennedy) [inline image_services/opera/metadata/metadata_text/OPERA_RTC_CA_VH_metadata.txt:17]: I suspect JPL should be added here...

Comment (jhkennedy) [reply image_services/opera/metadata/metadata_text/OPERA_RTC_CA_VH_metadata.txt:17]: And to all similar

Comment (jhkennedy) [reply image_services/opera/metadata/metadata_text/OPERA_RTC_CA_VH_metadata.txt:17]: On the About OPERA page, on the OPERA website, it's written like:
> JPL, along with project partners from NASA’s Goddard Space Flight Center, the U.S. Geological Survey (USGS), the University of Maryland College Park, the University of Alaska Fairbanks, and Southern Methodist University

So I'd write something like:
> The OPERA project is run/managed/? by JPL with partners from NASA’s Goddard Space Flight Center, the U.S. Geological Survey (USGS), the University of Maryland College Park, the University of Alaska Fairbanks, and Southern Methodist University

**Review by jhkennedy — DISMISSED** [review]
Looks good! I think it would be worth looking at how we describe the OPERA project partners, to emphasize that it's run out of JPL. I know we say it a few paragraphs before, but that context is easily lost by the time you're at the overall project makeup description again. 

Also a couple white-space suggestions so that files have a trailing newline that are entirely knitpicky.

**Review by jhkennedy — APPROVED** [review]

**Pull request merged**